.travis.yml

```diff
@@ -3,6 +3,23 @@ language: cpp
 jobs:
   include:
 
+    - &cbmc-viewer-stage
+      stage: Linter + Doxygen + non-debug Ubuntu/gcc-5 test
+      env: NAME="cbmc-viewer"
+      addons:
+        apt:
+          packages:
+            - python3
+            - python3-pip
+      install:
+        pip3 install lxml
+      script:
+        which goto-cc
+	which cbmc
```

Comment:
I'm not sure this will work - this change to the travis config adds a whole new job which doesn't do anything to actually build cbmc or its tools, so these tools won't be on the path. Depending on how long you expect it to take to run these new regression tests, it might be a better idea to integrate them into one of the other existing jobs as part of the standard regression tests?

Reply:
I am still figuring out how to write the regression tests.  I don't know how to debug this other than on an open PR.  I created a draft PR, but travis wouldn't run on a draft PR.

Reply:
We have done as you advised, and travis is now working.